src/vaults/vaults.utils.ts

```diff
@@ -555,10 +548,26 @@ export async function estimateVaultPerformance(
     throw new Error(`${vaultDefinition.name} does not have adequate harvest history`);
   }
 
-  const totalDuration = recentHarvests[0].timestamp - recentHarvests[data.length - 1].timestamp;
+  let totalDuration;
+
+  // TODO: generalize this for voting vaults + look up their voting periods
+  if (vaultDefinition.vaultToken === TOKENS.BVECVX) {
+    totalDuration = ONE_DAY_SECONDS * 14;
```

Comment:
If we set this to `1`, we wouldn't need to custom handle `measuredValue` further down in code.  We already have a custom handler for bveCVX when determining cutoff, so we can just do `recentHarvests[0].timestamp - VOTING_PERIOD.CVX` where `VOTING_PERIOD.CVX = ONE_DAY_SECONDS * 14`